common/python/crypto_utils/crypto_utility.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018 Intel Corporation
+# Copyright 2020 Intel Corporation
```

Comment:
Keep it as 2018. This is just file rename. Old functions in this file still hold copyright from 2018 ?

Reply:
I have retained utility.py and created this new file to group all crypto util functions. Hence copyright is updated to latest.